### PR TITLE
Added details structure

### PR DESCRIPTION
**Pull request by gzamudio**
### Summary
- Added details structure

### Screenshot
Captura de Pantalla 2019-09-13 a la(s) 19 17 55

### Trello Card
- https://trello.com/c/Fvotr4lC

### CODE REVIEW

File: scss/details.scss [hunk 1]
```diff
@@ -10,15 +16,23 @@
...
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
 }
 
 .property-container {
   margin: 14px 0;
   
   &:first-child {
+    border-bottom: 4px solid $earls_green;
     margin: 24px 0;
+    padding: 0 0 12px 0;
```
Comment (nvescio) [inline scss/details.scss]: I'd use `padding-bottom: 12px;` here

File: scss/details.scss [hunk 2]
```diff
@@ -10,15 +16,23 @@
...
-	font-size: 20px;
-	font-weight: bold;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.container {
+  width: 1000px;
+  height: 416px;
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
```
Comment (nvescio) [inline scss/details.scss]: Try to leave a space after the commas --> `rgba(0, 0, 0, 0.5)`

**Review by nvescio — CHANGES_REQUESTED** [review]

File: scss/details.scss [hunk 3]
```diff
@@ -10,15 +16,23 @@
...
 
 .book-prop {
-	font-size: 20px;
-	font-weight: bold;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.container {
+  width: 1000px;
```
Comment (BJordi) [inline scss/details.scss]: Sort it.

Comment (BJordi) [inline scss/details.scss]: Why not `width: 100%;`? :thinking:

**Review by BJordi — CHANGES_REQUESTED** [review]

**Review by nvescio — APPROVED** [review]

**Pull request merged**